lib/api/1.1/southbound/notification.js

```diff
@@ -33,7 +33,16 @@ function notificationRouterFactory (
 
     router.post('/notification/', rest(function (req) {
         var message = _.defaults(req.query || {}, req.body || {});
-        return notificationApiService.postNotification(message);
+
+        if (_.has(message, 'nodeId')) {
```

Comment:
Though the logic looks simple here. 
Will it be better to move it inside `notificationApiService` ? and expose a general `postNotification` function here ?